website/content/configuration/_advanced_ipaddresspool_configuration.md

```diff
@@ -48,6 +48,42 @@ To specify a single IP address in a pool, use `/32` in the CIDR notation
 (e.g. `42.176.25.64/32`).
 {{% /notice %}}
 
+### Reduce scope of address allocation to specific Namespace and Service
+
+This option can be used to reduce the scope of particular IPAddressPool
+to set of namespaces and services, by adding an optional namespace
+and/or service selectors.
+
+```yaml
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  name: ippool-ns-service-alloc-sample
+  namespace: metallb-system
+spec:
+  addresses:
+    - 192.168.20.0/24
+  avoidBuggyIPs: true
+  serviceAllocation:
+    priority: 50
+    namespaces:
+      - namespace-a
+      - namespace-b
+    namespaceSelectors:
+      - matchLabels:
+          foo: bar
+    serviceSelectors:
+      - matchExpressions:
+          - {key: app, operator: In, values: [bar]}
+```
+
+The above IPAddressPool example is pinned to service(s) with has a label
+matching expression `key: app, operator: In, values: [bar]` and created
+either in `namespace-a` or `namespace-b`.
+When a service explicitly chooses an IPAddressPool (or an IP Address
```

Comment:
Nit: add the note modifier here (just because the pr needs to be reworked anyway)

Reply:
done.

Reply:
what I meant is, use the `{{% notice note %}}` wrapper to highlight this.

Reply:
and since we are going to touch this, I'd rephrase with When a service explicitly chooses an IPAddressPool via the xxx annotation or an ip via the spec.loadbalancerip...